### ISSUE TITLE
[catboost4j-spark] The earlyStoppingRounds parameter is not recognized

### ISSUE THREAD

**Issue by candalfigomoro** [catboost/catboost#1649]
Problem:
With PySpark, when I set the earlyStoppingRounds parameter (e.g. `earlyStoppingRounds=10`), I get the following error:

```
py4j.protocol.Py4JJavaError: An error occurred while calling o318.fit.
: ai.catboost.CatBoostError: catboost/private/libs/options/plain_options_helper.cpp:495: Unknown option {early_stopping_rounds} with value "10"
	at ru.yandex.catboost.spark.catboost4j_spark.core.src.native_impl.native_implJNI.GetOneHotMaxSize(Native Method)
	at ru.yandex.catboost.spark.catboost4j_spark.core.src.native_impl.native_impl.GetOneHotMaxSize(native_impl.java:181)
	at ai.catboost.spark.CatBoostPredictorTrait.addEstimatedCtrFeatures(CatBoostPredictor.scala:55)
	at ai.catboost.spark.CatBoostPredictorTrait.addEstimatedCtrFeatures$(CatBoostPredictor.scala:43)
	at ai.catboost.spark.CatBoostClassifier.addEstimatedCtrFeatures(CatBoostClassifier.scala:362)
	at ai.catboost.spark.CatBoostClassifier.preprocessBeforeTraining(CatBoostClassifier.scala:400)
	at ai.catboost.spark.CatBoostPredictorTrait.fit(CatBoostPredictor.scala:152)
	at ai.catboost.spark.CatBoostPredictorTrait.fit$(CatBoostPredictor.scala:109)
	at ai.catboost.spark.CatBoostClassifier.fit(CatBoostClassifier.scala:362)
```

catboost version: catboost-spark_2.4_2.12-0.25

@andrey-khropov

**Comment by nvithayapale** [comment]
@andrey-khropov I have encountered the same issue that earlyStoppingRounds is not recognized in Spark Scala.

`User class threw exception: ai.catboost.CatBoostError: catboost/private/libs/options/plain_options_helper.cpp:495: Unknown option {early_stopping_rounds} with value "100" at ru.yandex.catboost.spark.catboost4j_spark.core.src.native_impl.native_implJNI.GetOneHotMaxSize(Native Method) at ru.yandex.catboost.spark.catboost4j_spark.core.src.native_impl.native_impl.GetOneHotMaxSize(native_impl.java:181) at ai.catboost.spark.CatBoostPredictorTrait.addEstimatedCtrFeatures(CatBoostPredictor.scala:55) at ai.catboost.spark.CatBoostPredictorTrait.addEstimatedCtrFeatures$(CatBoostPredictor.scala:43) at ai.catboost.spark.CatBoostRegressor.addEstimatedCtrFeatures(CatBoostRegressor.scala:222) at ai.catboost.spark.CatBoostPredictorTrait.preprocessBeforeTraining(CatBoostPredictor.scala:81) at ai.catboost.spark.CatBoostPredictorTrait.preprocessBeforeTraining$(CatBoostPredictor.scala:75) at ai.catboost.spark.CatBoostRegressor.preprocessBeforeTraining(CatBoostRegressor.scala:222) at ai.catboost.spark.CatBoostPredictorTrait.fit(CatBoostPredictor.scala:152) at ...`

**Comment by andrey-khropov** [comment]
Released in 0.26.
